load-tests/book_reviews_mutation/src/service.bal

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/graphql;
+import book_reviews_mutation.datasource as ds;
+import xlibb/pubsub;
+
+service /graphql on new graphql:Listener(9000) {
+    private final pubsub:PubSub pubsub;
+
+    function init() {
+        self.pubsub = new;
+    }
+
+    # A simple resource that returns a welcome message
+    # + return - greeting message
+    isolated resource function get geeting() returns string {
+        return "Welcome to Book Review Service";
+    }
+
+    # Add review to a book
+    # + input - represents a review input
+    # + return - the newly added Review
+    isolated remote function reviewAdd(readonly & ReviewInput input) returns Review|error {
+        ds:ReviewRow reviewRow = check ds:addReview(input);
+        check self.pubsub.publish("Reviews", reviewRow.cloneReadOnly(), timeout = -1);
```

Comment:
If we make the review a readonly type, we can omit this `cloneReadOnly` call as well.

Reply:
made types readonly with [2d4ed9a](https://github.com/ballerina-platform/module-ballerina-graphql/pull/1308/commits/2d4ed9a102749ec104813d2c82b2e4f214053bfd)